.github/workflows/haskell.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["9.6", "9.8"]
+        ghc: ["9.6", "9.8", "9.10"]
```

Comment:
If our next GHC target after 8.10 is 9.6, I'm wondering if we need both 9.8 and 9.10 here?